contracts/context-config/tests/sandbox.rs

```diff
@@ -1053,21 +1042,40 @@ async fn test_deploy() -> eyre::Result<()> {
         },
     };
     let signed = Signed::new(&request, |p| alice_cx_sk.sign(p))?;
-
+    // println!("signed {:?}", signed);
     let res = node1
         .call(&proxy_address, "mutate")
-        .args_json(json!(signed))
+        .args_json(signed.clone())
```

Comment:
`&T: Serialize` [where](https://docs.rs/serde/latest/serde/trait.Serialize.html#impl-Serialize-for-%26T) `T: Serialize`, so you should only need this:

```suggestion
        .args_json(&signed)
```